### PR TITLE
Check rpm version of underlying cluster for valid import

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#906
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py [hunk 1]
```diff
@@ -64,14 +65,34 @@ def run(self):
...
+                    "Failed to detect underlying cluster version"
+                )
+            lines = out.split('\n')
+            build_no = None
+            req_build_no = None
+            ver_det = lines[0].split('glusterfs-server-')[-1].split('.')
+            maj_ver = ver_det[0]
+            min_ver = ver_det[1]
+            rel = ver_det[2]
+            if '-' in rel:
```
Comment (r0h4n) [inline tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: Add a comment with sample of whatever the ideal output of `rpm -qa | grep glusterfs-server` is for future reference,

No one's gonna understand this after 2 months :)

Comment (shtripat) [reply tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py]: Ack

**Comment by shtripat** [comment]
Verified with https://github.com/Tendrl/monitoring-integration/pull/418 and https://github.com/Tendrl/node-agent/pull/779. Works as expected. But UI needs a fix for https://github.com/Tendrl/ui/issues/929 to display the correct version details in UI.

**Comment by r0h4n** [comment]
Please fix pep8

**Pull request merged**